### PR TITLE
JAMES-2551 Introduce Cassandra RabbitMQ product

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/container/guice/rabbitmq/src/main/java/org/apache/james/modules/rabbitmq/RabbitMQModule.java [hunk 1]
```diff
@@ -40,6 +63,43 @@
 
     @Override
     protected void configure() {
+        bind(Clock.class).toInstance(Clock.systemUTC());
+        bind(ThreadLocalRandom.class).toInstance(ThreadLocalRandom.current());
+        bind(CassandraMailQueueViewConfiguration.class).toInstance(CassandraMailQueueViewConfiguration
```
Comment (chibenwa) [inline server/container/guice/rabbitmq/src/main/java/org/apache/james/modules/rabbitmq/RabbitMQModule.java]: We would need a configuration file for this - separate pull request

**Comment by aduprat** [comment]
:+1:

**Comment by aduprat** [comment]
```
[ERROR] Errors: 
[ERROR]   CassandraRabbitMQJamesServerTest.mailsShouldBeWellReceived:77 » ConditionTimeout
```

**Comment by aduprat** [comment]
merged

**Pull request closed without merging**